app/src/main/java/com/example/bisit/ui/customerPay/TossPayActivity.kt

```
@@ -164,22 +170,106 @@ class TossPayActivity : AppCompatActivity() {
+                    override fun shouldOverrideUrlLoading(view: WebView?, request: WebResourceRequest?): Boolean {
+                        // 새 창에서의 URL 처리
+                        val url = request?.url?.toString() ?: return false
+                        Log.d(TAG, "Popup URL: $url")
+                        
+                        // 외부 앱 실행 로직 등 동일하게 적용 가능
+                        if (!url.startsWith("http") && !url.startsWith("javascript:")) {
+                             try {
+                                val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
+                                startActivity(intent)
+                                return true
+                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)